reactor-core/build.gradle

```diff
@@ -22,7 +22,7 @@ apply plugin: 'kotlin'
 
 ext {
 	bndOptions = [
-			"Export-Package": "!*internal*,reactor.*;-noimport:=true",
+			"Export-Package": "!*internal*,reactor.adapter.*,reactor.core.*,reactor.util.*;-noimport:=true",
```

Comment:
I am surprised that we need to duplicate it here o_O OSGI is the gift that keeps on giving 😂

Reply:
it otherwise complains about `reactor.shaded` which IIRC is linked to the wonky stubs stuff we did for `StackWalker`-and-al

Reply:
1) it only applies to `reactor-tools`, right?
2) what if we explicitly exclude `reactor.shaded.*`?

Reply:
re `StackWalker` - not the case, but `reactor-tools`' shading of ByteBuddy